apps/explorer/src/utils/amplitude.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { Types } from '@amplitude/analytics-browser';
+import { getProductAnalyticsConfig } from '@mysten/core';
+
+import { ampli } from './ampli';
+
+const IS_PROD_ENV = import.meta.env.PROD;
+
+export async function initAmplitude() {
+    const { disableCookies } = await getProductAnalyticsConfig(IS_PROD_ENV);
+    ampli.load({
+        environment: IS_PROD_ENV ? 'production' : 'development',
```

Comment:
I opted to not throw these values in `getProductAnalyticsConfig` so individual apps can have more fine-grained control of their specific settings without having to do overrides